starter_code/index.js

```diff
@@ -1 +1,13 @@
 const Elevator = require('./elevator.js');
+const Person = require('./person.js');
+
+let test = new Elevator();
+
+let luffy = new Person("Luffy", 1, 1);
+let zorro = new Person("Zorro", 1, 2);
+let sanji = new Person("Sanji", 2, 3 );
+let franky = new Person("Franky", 3, 4);
+
+
```

Comment:
You forgot to use the "call" method on your people, so the elevator actually runs :)

test.call(zorro) etc.